### ISSUE TITLE
Issue running `!icenet_data_sic` with given date range in 01.cli_demonstration.ipynb

### ISSUE THREAD

**Issue by bnubald** [icenet-ai/icenet-notebooks#12]
- IceNet version: 0.2.6
- Python version: 3.8.18
- Operating System: Windows (WSL2 Ubuntu 22.04.3 LTS)

### Description

Running 7th cell in [01.cli_demonstration.ipynb](https://github.com/icenet-ai/icenet-notebooks/blob/main/01.cli_demonstration.ipynb) causes FTP error.

`!icenet_data_sic south -d 2020-1-1 2020-1-30`

### What I did

After downloading data and masks in previous cells, when running the 7th cell with `!icenet_data_sic south -d 2020-1-1 2020-1-30` command, get the following warning and no file downloads:

```bash
[26-10-23 22:20:44 :INFO    ] - OSASIF-SIC Data Downloading
[26-10-23 22:20:44 :INFO    ] - Downloading SIC datafiles to .temp intermediates...
[26-10-23 22:20:45 :INFO    ] - FTP opening
[26-10-23 22:20:45 :WARNING ] - FTP error, possibly missing month chdir for 2020_01_01
[26-10-23 22:20:45 :WARNING ] - FTP error, possibly missing month chdir for 2020_01_02
[26-10-23 22:20:45 :WARNING ] - FTP error, possibly missing month chdir for 2020_01_03
[26-10-23 22:20:45 :WARNING ] - FTP error, possibly missing month chdir for 2020_01_04
[26-10-23 22:20:45 :WARNING ] - FTP error, possibly missing month chdir for 2020_01_05
[26-10-23 22:20:45 :WARNING ] - FTP error, possibly missing month chdir for 2020_01_06
[26-10-23 22:20:45 :WARNING ] - FTP error, possibly missing month chdir for 2020_01_07
[26-10-23 22:20:46 :WARNING ] - FTP error, possibly missing month chdir for 2020_01_08
[26-10-23 22:20:46 :WARNING ] - FTP error, possibly missing month chdir for 2020_01_09
[26-10-23 22:20:46 :WARNING ] - FTP error, possibly missing month chdir for 2020_01_10
```

**Comment by bnubald** [comment]
See related icenet-ai/icenet#192

**Comment by bnubald** [comment]
Fixed on latest v0.2.7_dev branch via https://github.com/icenet-ai/icenet/pull/200
